src/openrct2/object/Object.cpp

```diff
@@ -116,6 +116,16 @@ void rct_object_entry::SetName(const std::string_view& value)
     std::memcpy(name, value.data(), std::min(sizeof(name), value.size()));
 }
 
+std::vector<std::string> Object::GetAuthors()
+{
+    return _authors;
+}
+
+void Object::SetAuthors(std::vector<std::string> authors)
```

Comment:
I would pass an rvalue reference of authors no point having all the extra copies

Reply:
Thank you both for the reference advice, hadn't thought about that before